### PR TITLE
Added names of primitives, declared functions, wrote out functions as…

### PR DESCRIPTION
**Pull request by sbl-plugins**
… thought it was called with an argument.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -10,158 +10,162 @@ video: Chapter 2 - Part 3
...
+return { }; }
+
+int mailLetter(int envelope, float stamp, int mailbox, int letter){ ignoreUnused(envelope, stamp, mailbox, letter);
+return { }; }
+                                                                   
+double buyStamps(bool postOfficeOpen){ ignoreUnused(postOfficeOpen);
+return { }; }
+
+int cookDinner(int oven, int airFryer, int chicken, int brownRice, int chocolateCake, bool isHungry){ ignoreUnused(oven, airFryer, chicken, brownRice, chocolateCake, isHungry);
+return { }; }
```
Comment (matkatmusic) [inline main.cpp]: culy braces go on their own line with no other code
```
int funcName(int a, float b);
{
    ignoreUnused(a, b);
    return { };
}
```
- [x] revise, conform to the course coding standard

File: main.cpp [hunk 2]
```diff
@@ -10,158 +10,162 @@ video: Chapter 2 - Part 3
...
+        char knob = 'H';
+        char view = 's';
+
+        unsigned int blackPants = 7;
+        unsigned int greenJackets = 5;
+        unsigned int shoulderPads = 567;
+
+        ignoreUnused(shoes, violins, ports, height, weight, temperature, areOpen, canBeRemoved, isVisible, bagels, creamCheese, coffeeCups, grade,         knob, view, blackPants,greenJackets, shoulderPads);
+
+    }
```
Comment (matkatmusic) [inline main.cpp]: -[ ] revise, all of this is indented to the right 4 more spaces than it should be.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Seems like your automatic formatter created some problems with the formatting of the code in relation to the course coding standard.

File: main.cpp (hunk 2 above)
Comment (sbl-plugins) [reply main.cpp]: Corrected.

File: main.cpp (hunk 1 above)
Comment (sbl-plugins) [reply main.cpp]: Completed to the point that it runs, but don't understand why to remove the ";" at the end of the lines where I declare the variables. For example: 

_float calculateStatus(float weight, float height, float temperature, int active, int **orientation)**
{
    ignoreUnused(weight, height, temperature, active, orientation);
    return { };
}_

 -- semicolon removed after "**orientation)**" and others. In your example above, the semicolon is present.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**